app/components/reasons_for_rejection_component.html.erb

```diff
@@ -58,6 +58,7 @@
           <% end %>
         </li>
       <% end %>
+      To check the course requirements view the course <%= link_to 'here.', "https://www.find-postgraduate-teacher-training.service.gov.uk/course/#{application_choice.provider.code}/#{application_choice.course.code}#section-entry", class: "govuk-link" %>
```

Comment:
you can use `<%= govuk_link_to `'here.', "#{@application_choice.course.find_url}#section-entry" %> here

Reply:
So at the moment, this component is being called in quite a few places. We only want to render this link on the application dashboard. Let's discuss 👍